samples/guide/src/main/java/okhttp3/recipes/kt/DevServer.kt

```diff
@@ -114,7 +51,7 @@ class DevServerAndroid {
   val hosts = arrayOf(server.hostName)
 
   val platformTrustManager = platformTrustManager()
-  val trustManager = AndroidAllowlistedTrustManager(platformTrustManager, *hosts)
+  val trustManager = OkHttpTrustManager.hostOverride(platformTrustManager, "localhost", InsecureTrustManager)
   val sslSocketFactory = Platform.get().newSSLContext().apply {
```

Comment:
This sample should be cleaned up to not use Platform.

Reply:
@swankjesse How can we avoid Platform here and dealing with sslSocketFactory.  Just using OkHttpTrustManager and heldCertificates.

Should OkHttpTrustManager be keyManager aware?

Reply:
It is really convenient to have all the stuff you need for an SSLContext all in one place. Though the API is still tricky to get right.